packages/design-system-docs/src/pages/components/Tooltip/_Tooltip.docs.scss

```diff
@@ -53,14 +53,10 @@ Style guide: components.tooltip.tooltip-icon
 
 The following Sass variables can be overridden to theme choice fields:
 
```

Comment:
```suggestion
```

Reply:
Need to remove that extra line as with the extra line, subsequent variables are not being displayed by the docs site.